Webreidi hackathon2022 (#342)
NuevoFoundation/workshops: content/english/python-basics/data-types/_index.md

---
title: "Data Types"
draft: false
weight: 2
---

## Overview

> In programming, data type is an important concept. Variables can store data of different types, and different types can do different things.

Reviewer: ```suggestion
In programming, data types are an important concept. Variables can store data of different types, and different types can do different things.
```
